docs/changelog.md

```diff
@@ -5,6 +5,21 @@ toc:
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.125 (2023-04-06)
+
+### Features
+
+- Added the [required-min-length-string-type-property](./rules/required-min-length-string-type-property.md) rule for required property in the API definition with type `string` should have a `minLength`. 
+
+### Fixes
+
+- Fixed an issue with `$ref` in path parameters during for the `join` command.
+- Other internal stability fixes and improvements.
+
+### Changes
+
+- Changed arguments for assertion custom functions. Now as third parameters instead of `location` it has `ctx` object. See [Custom functions](./rules/custom-rules.md#custom-function-example).  
```

Comment:
```suggestion
- Changed the arguments for assertion custom functions. Now as the third parameter, instead of a `location` object, assertion custom functions include a `ctx` object. See [Custom functions](./rules/custom-rules.md#custom-function-example).  
```